src/Cli/dotnet/CommonOptions.cs

```diff
@@ -281,10 +281,10 @@ internal static CliArgument<T> AddCompletions<T>(this CliArgument<T> argument, F
 
     public enum VerbosityOptions
     {
-        quiet,
-        q,
         minimal,
```

Comment:
Thanks for the contribution! @baronfel Do you think it might be ok with change the default verbosity here in `CommonOptions`?

Reply:
I'm a little concerned about the enum reordering - that has implications for code like `default(VerbosityOptions)`, that would make me want to inspect this _very closely_.

Reply:
> I'm a little concerned about the enum reordering - that has implications for code like `default(VerbosityOptions)`, that would make me want to inspect this _very closely_.

From the documentation, the `default(VerbosityOptions)` is supposed to be `minimal` anyway. This is more like a bug fix rather than feature/behavior fix.
![image](https://github.com/dotnet/sdk/assets/32099214/64dbde56-5a3d-4c54-b76c-8cfb0e411c1a)

Reply:
> I'm a little concerned about the enum reordering - that has implications for code like `default(VerbosityOptions)`, that would make me want to inspect this _very closely_.

As a alternative, we may change the default verbosity for `global install` only? This will not affect default verbosity options across the entire sdk.

Reply:
That sounds a great idea. Can you try changing the default verbosity for `tool install` command for global tools?

Reply:
> That sounds a great idea. Can you try changing the default verbosity for `tool install` command for global tools?

Thanks. Changed it to only default verbosity to tool install